doc/plugin_tutorial/README.md

```diff
@@ -3,13 +3,15 @@ How to write plugins in Coq
   # Working environment
 
   In addition to installing OCaml and Coq, it can help to install several tools for development.
-  
-  ## Merlin
+
+  ## Tuareg and Merlin
 
   These instructions use [OPAM](http://opam.ocaml.org/doc/Install.html)
 
 ```shell
 opam install merlin               # prints instructions for vim and emacs
+opam user-setup install           # automatically configures editors for merlin
+opam install tuareg               # syntax highlighting for OCaml
```

Comment:
Lines have to be inverted here; `user-setup` should be ran once all the addons are done.

This line will setup mlg:
```
(add-to-list 'auto-mode-alist '("\\.mlg$"      . tuareg-mode) t)
```
I have many other useful recommendations, but instead of adding them here, we should unify the recommended setup in `dev/doc/README.md` [and move the wiki part there]

Reply:
Is that a move you want to make immediately (rather than merging this PR) or would you rather merge this first, and then tackle that later?

Reply:
(I've inverted the lines)

Reply:
As you prefer, I think the move would be great [and anyways the information in the plugin-tuto readme is quite incomplete IMHO], so if you have time to do it let's go, otherwise we can just merge this for now.

Reply:
I think I'd rather punt on that, since I don't have much time right now. So please evaluate this as-is and we can make another issue for unified development setup documentation